inst/tests/tests.Rraw

```diff
@@ -9977,7 +9977,7 @@ test(1658.31, fwrite(ok_dt, qmethod=c("double", "double")), error="length(qmetho
 test(1658.32, fwrite(ok_dt, col.names="foobar"), error="isTRUEorFALSE(col.names)")
 
 # null data table (no columns)
-test(1658.33, fwrite(data.table(NULL)), NULL, warning="Nothing to write")
+test(1658.33, fwrite(data.table(NULL)), "", warning="Nothing to write")
```

Comment:
Hmm, wonder how useful this case is too -- when `fwrite()` to the terminal, maybe `NULL` should be returned as well? Not sure any inconsistency is worthwhile for this case though.